plugins/rds/rdsAutomatedBackups.js

```diff
@@ -49,6 +49,11 @@ module.exports = {
 				// For resource, attempt to use the endpoint address (more specific) but fallback to the instance identifier
 				var db = describeDBInstances.data[i];
 				var dbResource = db.DBInstanceArn;
+				debugger;
+				// skip if it is read only replica Source Indentifier
+				if ('ReadReplicaSourceDBInstanceIdentifier' in db){
+					continue;
+				}
```

Comment:
Can you change this to `if (db.ReadReplicaSourceDBInstanceIdentifier)`? The AWS API may set this property, even if it is `null`.